components/web/components/org.wso2.carbon.apimgt.publisher.commons.ui/src/main/fragments/api-client-js/public/js/lib/bootstrap-editable.js

```diff
@@ -0,0 +1,6807 @@
+/*! X-editable - v1.5.1 
```

Comment:
Moved the library to a common fragment - 52f15bce21dc13955d1d57d660143c4f3f7a2236